pkg/apis/gitwatcher.cattle.io/v1/types.go

```diff
@@ -31,8 +31,10 @@ type GitWatcherSpec struct {
 	Provider                       string            `json:"provider,omitempty"`
 	Push                           bool              `json:"push,omitempty"`
 	PR                             bool              `json:"pr,omitempty"`
-	Tag                            bool              `json:"tag,omitempty"`
 	Branch                         string            `json:"branch,omitempty"`
+	Tag                            bool              `json:"tag,omitempty"`
+	TagQuery                       string            `json:"tagQuery,omitempty"`
```

Comment:
What about TagInclude or TagIncludeRegexp and TagExcludeRegexp.  Just seems like the two regexp's should be similar in name.

Reply:
Or TagRegexp, TagExcludeRegexp.

Reply:
updated